### DISCUSSION TITLE
Comparison of risingwave, apache flink, and materiaze

### DISCUSSION THREAD

**Discussion by liurenjie1024** [ArroyoSystems/arroyo#52] [General]
Could you provide a comparison with other new streaming processing systems like risingwave, apache flink and materialize?

**Comment by breezykermo** [comment]
There are some answers to this now on the Arroyo blog, for example:
- https://www.arroyo.dev/blog/why-not-flink
- https://www.arroyo.dev/blog/how-arroyo-beats-flink-at-sliding-windows

**Comment by Doslin** [comment]
Here's a blog you might be interested in
- https://risingwave.com/blog/top-7-apache-flink-alternatives-a-deep-dive/
